core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java

```diff
@@ -105,6 +116,7 @@ static Backoff random(long minDelayMillis, long maxDelayMillis, Supplier<Random>
      * <ul>
      *   <li>{@code exponential=200:10000:2.0,jitter=0.2} (default)</li>
      *   <li>{@code exponential=200:10000,jitter=0.2,maxAttempts=50} (multiplier omitted)</li>
+     *   <li>{@code fibonacci=200:10000,jitter=0.2,maxAttempts=50}</li>
```

Comment:
Could you add the parsing logic in `BackoffSpec` and add the test case for it in `BackoffSpecTest`, please?

Reply:
Thanks for the review, I will add it as well!